src/figuresClasses.ts

```diff
@@ -1,19 +1,85 @@
-export interface Figure {
-
+enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle'
 }
 
-export class Triangle {
+enum Color {
+  red = 'red',
+  green = 'green',
+  blue = 'blue'
+}
 
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea: () => number;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  public shape: Shape = Shape.triangle;
```

Comment:
```suggestion
  shape = Shape.triangle;
```
If you follow suggestion above, you can assign this way. 
---
You don't need `public` here. `public` is default. In `constructor` we write it because otherwise we have to assign it in the `constructor`.

Reply:
Thank you Yasar.  I get it now.